bitcoin/src/lib.rs

```diff
@@ -13,21 +13,25 @@
 //! See README.md for detailed documentation about development and supported
 //! environments.
 //!
-//! ## Available feature flags
+//! # Features
```

Comment:
Perhaps it should be "Cargo features" to distinguish it with "general functionality provided by the crate".